<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurfaceExtractorViewControls</class>
 <widget class="QWidget" name="SurfaceExtractorViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>315</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <widget class="QWidget" name="wgtControls" native="true">
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
      <item row="0" column="0">
       <widget class="QLabel" name="lblReferenceImageLabel">
        <property name="text">
         <string>Reference image:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblReferenceImage">
        <property name="text">
         <string>No image selected.</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblTargetReduction">
        <property name="text">
         <string>Target reduction:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lblMaxNumberOfPolygons">
        <property name="text">
         <string>Max. number of polygons:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="spbMaxNumberOfPolygons"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblGaussianSmooth">
        <property name="text">
         <string>Apply smoothing:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblGaussianStdDev">
        <property name="text">
         <string>Standard deviation:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="cbxGaussianSmooth">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="spbGaussianStdDev"/>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="spbTargetReduction"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblThreshold">
        <property name="text">
         <string>Threshold:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="spbThreshold"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblSurfaceImageLabel">
        <property name="text">
         <string>Surface:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lblSurfaceImage">
        <property name="text">
         <string>No surface selected.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>52</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnCreate">
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <tabstops>
  <tabstop>cbxGaussianSmooth</tabstop>
  <tabstop>spbGaussianStdDev</tabstop>
  <tabstop>spbTargetReduction</tabstop>
  <tabstop>spbMaxNumberOfPolygons</tabstop>
  <tabstop>btnCreate</tabstop>
  <tabstop>btnApply</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
